contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.0.0--2.1.0.sql

```diff
@@ -3,6 +3,31 @@
  
 SELECT set_config('search_path', 'sys, '||current_setting('search_path'), false);
 
+-- Drops a view if it does not have any dependent objects.
+-- Is a temporary procedure for use by the upgrade script. Will be dropped at the end of the upgrade.
+-- Please have this be one of the first statements executed in this upgrade script. 
+CREATE OR REPLACE PROCEDURE babelfish_drop_deprecated_view(schema_name varchar, view_name varchar) AS
```

Comment:
Just for reference sake...

I'm very tempted to say we should define this function in pg_temp, so that we ensure it can't be permanent. However, there's one significant gotcha to doing that: the extension will be marked as depending on this function, so if the function is sitting in pg_temp when the session disconnects, the forcible removal will actually drop the extension. Ultimately I wish there was a better way to safeguard/ensure that this function will actually be dropped at the end of the script...